### PR TITLE
fix: db migration rename task id to UUID

### PR DESCRIPTION
**Pull request by safeamiiir**
DB on Aqueduct-core's been migrated and some more info has been added to the README file.

### CODE REVIEW

File: alembic/README [hunk 1]
```diff
@@ -8,9 +8,9 @@ The following steps should be followed in the Python environment configured for
 
 > **NOTE:** Always assume the database is populated with existing data!
 
-1. Make sure the environment variables with the database connection details are loaded into the environment.
+1. Make sure the environment variables with the database connection details are loaded into the environment. We're using Azure, and we additionally need to make sure it's using the same environment variables to be connected to the database on Azure, as that's in the current state and does not require changes, but to make sure about that, we need to do the next step check.
```
Comment (str-anger) [inline alembic/README:11]:
```suggestion
1. Make sure the environment variables with the database connection details are loaded into the environment. If you are using Azure instance, you will additionally need to make sure it's using the same environment variables to be connected to the database on Azure, as that's in the current state and does not require changes, but to make sure about that, we need to do the next step check.
```

The fact we are using azure is our personal fact. This is not related to open-source repo.

**Review by str-anger — APPROVED** [review]
Conditionally ok, but text may be slightly different. Also, how this happened table name was different?

**Comment by safeamiiir** [comment]
> Conditionally ok, but text may be slightly different. Also, how this happened table name was different?

It was a typo with the table name, missed because we didn't actually test it.

File: alembic/README [hunk 2]
```diff
@@ -8,9 +8,9 @@ The following steps should be followed in the Python environment configured for
 
 > **NOTE:** Always assume the database is populated with existing data!
 
-1. Make sure the environment variables with the database connection details are loaded into the environment.
+1. Make sure the environment variables with the database connection details are loaded into the environment. If you are using Azure instance, you will additionally need to make sure it's using the same environment variables to be connected to the database on Azure, as that's in the current state and does not require changes, but to make sure about that, we need to do the next step check.
 
-2. Make sure the database is up to date with the head of changes. (If it is not this step tries to run the migration one by one):
+2. As an initial check, make sure the database is up to date with the head of changes. (If it is not this step, try to run the migration one by one):
```
Comment (jatinriverlane) [inline alembic/README:13]:
```suggestion
2. As an initial check, make sure the database is up to date with the head of changes. (If this step fails, try to run the migration one by one):
```

**Review by jatinriverlane — APPROVED** [review]
Thanks for fixing this @safeamiiir
